src/background.js

```diff
@@ -90,6 +90,7 @@ const stateManager = {
 
   async rememberDisableHeuristics() {
     log("Remembering to never run heuristics again");
+    browser.experiments.preferences.clearUserPref(DOH_SELF_ENABLED_PREF);
```

Comment:
Is this the right place to do this? I'd prefer clearing this pref in the code that actually handles the event. We don't want to make a function called `rememberDisableHeuristics` responsible for more than its job description.

Reply:
@nhnt11 I don't think it is after consideration. I moved the clear pref call to occur where that function is called, rather than add it to the function. Thanks!

Reply:
The new call site looks good, thanks!